loopd/swapclient_server.go

```diff
@@ -1111,7 +1111,7 @@ func validateLoopOutRequest(ctx context.Context, lnd lndclient.LightningClient,
 		return 0, err
 	}
 
-	channels, err := lnd.ListChannels(ctx)
+	channels, err := lnd.ListChannels(ctx, false, false)
```

Comment:
Do we need to bump lndclient version if we're always passing in false,false here?